### PR TITLE
change color of secondary links to lighter gray

### PR DESCRIPTION
**Pull request by Dinika**
User-Story: [INSPIR-1415](https://its.cern.ch/jira/browse/INSPIR-1415)

Screenshot
screenshot from 2018-10-05 11-34-52

Signed-off-by: Dinika Saxena <dinika.saxena@cern.ch>

### CODE REVIEW

**Comment by michamos** [comment]
Personally, I still think they stand out too much. I also think it's confusing to have two different representations for links (blue and underlined), with no clear pattern behind their use.

**Comment by drjova** [comment]
the idea behind this pattern is primary (blue) and secondary (grey), the most important information is blue and the rest grey

File: src/App.scss [hunk 1]
```diff
@@ -1,5 +1,10 @@
 .__App__ {
   min-height: 100vh;
+  %secondary {
```
Comment (harunurhan) [inline src/App.scss]: .secondary-link

File: src/App.scss [hunk 2]
```diff
@@ -11,8 +16,16 @@
...
   .mb4-important {
     margin-bottom: 4px !important;
   }
+  .secondary-item {
+    @extend %secondary;
+  }
+  .secondary-item-container {
+    font-style: italic;
+    a {
+      @extend %secondary;
```
Comment (harunurhan) [inline src/App.scss]: extend secondary link

File: src/App.scss [hunk 3]
```diff
@@ -11,8 +16,16 @@
       color: $link-hover-color;
     }
   }
-
   .mb4-important {
     margin-bottom: 4px !important;
   }
+  .secondary-item {
```
Comment (harunurhan) [inline src/App.scss]: unnecessary

File: src/App.scss [hunk 4]
```diff
@@ -11,8 +16,16 @@
...
+    @extend %secondary;
+  }
+  .secondary-item-container {
```
Comment (harunurhan) [inline src/App.scss]: `.secondary-container`

**Pull request merged**